libs/community/tests/integration_tests/retrievers/document_compressors/test_base.py

```diff
@@ -1,4 +1,5 @@
 """Integration test for compression pipelines."""
+import pytest
```

Comment:
```suggestion
```

Reply:
if I delete this annotation, the test is not run